apps/web/app/components/dashboard/dashboard-ui.css

```
@@ -0,0 +1,623 @@
+.dash-shell {
+  display: flex;
+  height: 100vh;
+  background-color: #0e0d0c;
+  color: #f5f2ee;
+  overflow: hidden;
+}
+
+.sidebar {
+  --sidebar-expanded: 280px;
+  --sidebar-collapsed: 72px;
+  width: var(--sidebar-expanded);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)